controllers/apply/forms/reaching-communities-form.js

```diff
@@ -203,40 +225,44 @@ formModel.registerSuccessStep({
              * Render a Nunjucks template to a string and
              * convert the string to inline HTML
              */
-            app.render('emails/applicationSummary', {
-                summary: summary,
-                form: formModel,
-                data: flatData
-            }, (errorRenderingTemplate, html) => {
-
-                if (errorRenderingTemplate) {
-                    return reject(errorRenderingTemplate);
-                }
+            app.render(
```

Comment:
Every change to this file below this line is a `prettier`/whitespace tweak, not a proper change.